packages/realm/src/app-services/Sync.ts

```diff
@@ -82,13 +56,16 @@ export class Sync {
   /** @deprecated Please use named imports */
   static NumericLogLevel = NumericLogLevel;
 
+  /** @deprecated Please use Realm.setLogLevel. Will be removed in v13.0.0. */
```

Comment:
Mainly changed adding a `@link` so that users can navigate to the new type. (Swapped the two sentences just for consistency 👍 )
```suggestion
  /** @deprecated Will be removed in v13.0.0. Please use {@link Realm.setLogLevel}. */
```